scripts/commit_classifier.py

```diff
@@ -294,7 +294,9 @@ def apply_phab(self, hg, phabricator_deployment, diff_id):
             # Stop as soon as a base revision is available
             if self.has_revision(hg, patch.base_revision):
                 logger.info(
-                    f"Stopping at diff {patch.id} and revision {patch.base_revision}"
+                    "Stopping at diff {} and revision {}".format(
+                        patch.id, patch.base_revision
+                    )
```

Comment:
The goal of the issue is to move from "f-string" to "%s formatting", not to `str.format`. See the reason here: https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/logging-fstring-interpolation.html

If you are in doubt about what the type is, you could execute the script to check. If it is hard to determine, `%s` should be okay in most cases.